### ISSUE TITLE
new option --progress for occ system:cron

### ISSUE THREAD

**Issue by DeepDiver1975** [owncloud/docs#1955]
see https://github.com/owncloud/core/pull/36304

The newly added option --progress/-p will show the progress bar. Without there will be no outpur

**Comment by mmattel** [comment]
Info: https://github.com/owncloud/docs/pull/1234 (Document the system:cron command) must be merged before another PR can be filed on top, as this one here adds content to the referenced one. Can´t be integrated to #1234 because a new oC release must be published containing the core fix.

@settermjd fyi

**Comment by mmattel** [comment]
Note: to fix that, I propose to take a look at https://github.com/owncloud/docs/blob/master/modules/admin_manual/pages/configuration/server/occ_command.adoc#the-filesscan-command in particular the `-q --quiet` option. Content can be copy/pasted and adopted reducing time needed to add this option properly and inline to the other occ commands.

**Comment by settermjd** [comment]
Will implement this, just as soon as #1234 is merged and backported.

**Comment by DeepDiver1975** [comment]
> @DeepDiver1975 any plan about a core (fix) release ?

10.3.1 or 10.4 - we will see ... @micbar

**Comment by micbar** [comment]
This will go in 10.3.1

**Comment by settermjd** [comment]
Thanks, @micbar.

**Comment by settermjd** [comment]
> Info: #1234 (Document the system:cron command) must be merged before another PR can be filed on top, as that one adds content to the referenced one. Can´t be integrated to #1234 because a new oC release must be published containing the core fix.
> 
> @settermjd fyi
> @DeepDiver1975 any plan about a core (fix) release ?

This has now been merged.
